syncer/syncer.go

```diff
@@ -1248,7 +1246,7 @@ func (s *Syncer) syncDML(tctx *tcontext.Context, queueBucket string, db *DBConn,
 				successF()
 				clearF()
 			}
-		case <-ticker.C:
+		case <-time.After(waitTime):
```

Comment:
maybe try ticker.Reset before `select`.

`time.After` will generate a new NewTimer every time, which is not good to performance in fact.